transcriptic/config.py

```diff
@@ -133,9 +132,10 @@ def __init__(
         self._rsa_key = None
         self._rsa_key_path = None
         self._rsa_secret = None
-        # Set/Load rsa_key from arguement as string or Path
+        # Set/Load rsa_key from argument as string or Path
         self.rsa_key = rsa_key
 
+        self.organization_id = organization_id
```

Comment:
would prefer to move this under the comment block below (which mentions setting session headers)

Reply:
Done